zerver/management/commands/create_user.py

```diff
@@ -78,7 +83,8 @@ def handle(self, *args, **options):
                 full_name = input("Full name: ")
 
         try:
-            notify_new_user(do_create_user(email, initial_password(email),
+            pw = options.get('password', initial_password(email))
+            notify_new_user(do_create_user(email, pw,
```

Comment:
A use case I had in mind: deploy 100+ Zulip servers with passwords derived from 1 master key. There is a limit of the number of realms that can be contained in a server in a box, so a way to scale is to have multiple servers.